bigchaindb/util.py

```diff
@@ -109,8 +110,7 @@ def serialize(data):
         str: JSON formatted string
 
     """
-    return json.dumps(data, skipkeys=False, ensure_ascii=False,
-                      separators=(',', ':'), sort_keys=True)
+    return rapidjson.dumps(data, skipkeys=False, ensure_ascii=False, sort_keys=True)
```

Comment:
Does `rapidjson.dumps()` assume the equivalent of `separators=(',', ':')` by default?